eos-tech-support/eos-download-image

```diff
@@ -96,17 +132,22 @@ def main():
     args = p.parse_args()
     base = INTERNAL_BASE_URL if args.internal else PUBLIC_BASE_URL
 
+    # Create the output directory if necessary
+    if args.outdir:
+        os.makedirs(args.outdir, exist_ok=True)
+
     if args.windows_tool:
-        fetch_windows_tool(base)
+        fetch_windows_tool(args, base)
     else:
         fetch_image(args, base)
 
 
-def fetch_windows_tool(base):
+def fetch_windows_tool(args, base):
     filename = 'endless-installer.exe'
+    dest = os.path.join(args.outdir, filename)
     url = '{}/endless-installer/{}'.format(base, filename)
     log("Fetching Windows Installer")
-    check_call("wget", "--timestamping", url)
+    check_call("wget", "--timestamping", "-O", dest, url)
```

Comment:
I see this warning from `wget`:

```
$ wget --timestamping -O /home/wjt/fnde/endless-installer.exe https://d1anzknqnc1kmb.cloudfront.net/endless-installer/endless-installer.exe
WARNING: timestamping does nothing in combination with -O. See the manual
for details.
```

`--timestamping` is used here so that if the installer has changed since it was last downloaded, it is updated. I think here you want:

```python
check_call("wget", "--timestamping", "--directory-prefix", args.outdir, url)
```

(Sorry, I didn't check this path in the last round of review.)